Dockerfile.buildx

```diff
@@ -1,9 +1,12 @@
 ARG GO_VERSION=1.12.10
-FROM golang:${GO_VERSION}-stretch
+ARG BUILDX_COMMIT=v0.3.0
 ARG BUILDX_REPO=https://github.com/docker/buildx.git
+
+FROM golang:${GO_VERSION}-stretch
+ARG BUILDX_REPO
 RUN git clone "${BUILDX_REPO}" /buildx
 WORKDIR /buildx
-ARG BUILDX_COMMIT=master
+ARG BUILDX_COMMIT
```

Comment:
Side note: docs at https://github.com/moby/moby/issues/37345#issuecomment-400245466 helped me to review this :)))

Reply:
Funny you reference it as "docs". I should try to find some time to incorporate it in the docs though